app-games/src/main/java/com/aptoide/android/aptoidegames/feature_payments/presentation/PreSelectedPaymentMethod.kt

```diff
@@ -0,0 +1,90 @@
+package com.aptoide.android.aptoidegames.feature_payments.presentation
+
+import androidx.compose.runtime.Composable
+import androidx.lifecycle.ViewModel
+import cm.aptoide.pt.extensions.runPreviewable
+import com.appcoins.payments.manager.presentation.PaymentMethodsUiState
+import dagger.hilt.android.lifecycle.HiltViewModel
+import javax.inject.Inject
+
+@HiltViewModel
+class PreSelectedPaymentMethodViewModel @Inject constructor(
+  //private val repository: PreSelectedPaymentMethodRepository,
+) : ViewModel() {
+  /*val uiState = repository.getPreselectedPaymentMethod()
+    .stateIn(
+      viewModelScope,
+      SharingStarted.Eagerly,
+      null
+    )
+
+  fun setSelection(id: String?) {
+    viewModelScope.launch {
+      repository.setPreselectedPaymentMethod(id)
+    }
+  }*/
+}
+
+@Composable
+fun PreselectedPaymentMethodEffect(
+  paymentMethodsState: PaymentMethodsUiState,
+  navigate: (String) -> Unit,
+) = runPreviewable(
+  preview = { },
+  real = {
+    /*var navigated by rememberSaveable { mutableStateOf(false) }
+    val vm = hiltViewModel<PreSelectedPaymentMethodViewModel>()
+    val preselectedId by vm.uiState.collectAsState()
+
+    LaunchedEffect(key1 = preselectedId, key2 = paymentMethodsState, key3 = navigated) {
+      if (paymentMethodsState is Idle && !navigated) {
+        paymentMethodsState.paymentMethods.find { it.id == preselectedId }
+          ?.getRoute(isPreSelected = true)
+          ?.also {
+            navigate(it)
+            navigated = true
+          }
+      }
+    }*/
+  }
+)
+
+@Composable
+fun rememberHasPreselectedPaymentMethod(): Boolean = runPreviewable(
+  preview = { false },
+  real = {
+    false
+    /*val vm = hiltViewModel<PreSelectedPaymentMethodViewModel>()
+    val uiState by vm.uiState.collectAsState()
+    uiState != null*/
+  }
+)
+/*
```

Comment:
If it is not used, then remove please.
If it will be used, no need to comment it out.

Reply:
fixed